inlang/source-code/cli/src/commands/machine/translate.ts

```diff
@@ -52,6 +52,7 @@ export async function translateCommandAction(args: { project: InlangProject }) {
 			log.error(`No inlang project found`)
 			return
 		}
+		const ffAliases = args.project.settings().featureFlags?.aliases
```

Comment:
had to double check what `ff` is

```suggestion
		const aliasesFeatureFlag = args.project.settings().featureFlags?.aliases
```

Reply:
👍 
Agree - will fix across both files.

Reply:
btw what about calling feature flags `experimental`? aligns with communication of other apps that use feature flags e.g. node cli, github etc. 

```diff
-aliasesFeatureFlag
+experimentalAliases
```

```diff
-args.project.settings().featureFlags.aliases
+args.project.settings().experimental.aliases
```

this is a naming discussion. no large impact expected. hence, either name it feature flags or experimental. both will be fine :)